gizmo.py

```diff
@@ -0,0 +1,56 @@
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+
+    def speak(self):
+        print (self.name)
+
+def hello(name, country = "Finland"):
+    print("Hello " + name + ", how are things in " + country + "?")
```

Comment:
You can make this prettier by using an f-string here (like you do below for the `relative_path` function)